### DISCUSSION TITLE
The latest release was 6 months ago

### DISCUSSION THREAD

**Discussion by isle9** [jamiebrynes7/obsidian-todoist-plugin#111] [General]
Is this plugin still being worked on?

**Comment by jamiebrynes7** [comment]
Hey @isle99, I'm not actively developing features on it at the moment. If there are critical bugs, I'll fix those, but some other things have taken precedence for my free time! 

PRs for features very welcome however!

**Reply by isle9** [reply]
Do you ming at least fixing the context menu bug or at least making a quick fix so that we can still open the tasks from within Obsidian somehow?

Just a link of any sorts to the task would be fine.

**Reply by jamiebrynes7** [reply]
Yup, I'll try to take a look this weekend!

**Reply by isle9** [reply]
Great, I remember that they changed something which broke the context menu a while ago. Also, you might want to check this new plugin [Extended Context Menu](https://github.com/phibr0/obsidian-extended-context-menu).

**Reply by jamiebrynes7** [reply]
Had a quick look at what was going wrong, was a pretty quick fix so I just went ahead and fixed that. Should be working again in `1.7.1`!

**Reply by kmaustral** [reply]
> Had a quick look at what was going wrong, was a pretty quick fix so I just went ahead and fixed that. Should be working again in `1.7.1`!

And if you get some time, it would be awesome to have a setting that allowed us to use Todoist as a Quick Capture tool, which Obsidian is lacking currently. It would mean being able to complete the tasks after they were downloaded.

**Reply by isle9** [reply]
> > Had a quick look at what was going wrong, was a pretty quick fix so I just went ahead and fixed that. Should be working again in `1.7.1`!
> 
> And if you get some time, it would be awesome to have a setting that allowed us to use Todoist as a Quick Capture tool, which Obsidian is lacking currently. It would mean being able to complete the tasks after they were downloaded.

A plugin to just that called Quick Capture IIRC should come out soon. It's currently being reviewed.

**Reply by isle9** [reply]
> Had a quick look at what was going wrong, was a pretty quick fix so I just went ahead and fixed that. Should be working again in `1.7.1`!

I have just one more small request. Can you maybe take a look why the plugin doesn't work on mobile? There's an issue for that already and someone seemed to have worked out what's the problem. This isn't urgent, but it would be nice to get it working on mobile eventually too.

**Comment by kmaustral** [comment]
Great news!
